lms/djangoapps/certificates/api.py

```diff
@@ -155,6 +146,19 @@ def get_certificate_for_user(username, course_key, format_results=True):
         return cert
 
 
+def get_certificate_for_user_id(user, course_id):
+    """
+    Retrieve certificate information for a user in a specific course.
+
+    Arguments:
+        user (User): A Django User.
+        course_id (int): Course ID
```

Comment:
The type for `course_id` doesn't look right here.

Reply:
Good catch, I will fix that.